_posts/01-04-01-Mac-Setup.md

```diff
@@ -29,14 +29,14 @@ MacPorts supports pre-compiled binaries, so you don't need to recompile every
 dependencies from the source tarball files, it saves your life if you don't
 have any package installed on your system.
 
-At this point, you can install `php53`, `php54`, `php55` or `php56` using the `port install` command, for example:
+At this point, you can install `php54`, `php55`, `php56` or `php70` using the `port install` command, for example:
 
-    sudo port install php54
-    sudo port install php55
+    sudo port install php56
+    sudo port install php70
```

Comment:
I checked, this actually exists. https://www.macports.org/ports.php?by=library&substr=php70